tools/cli/internal/cli/changelog/create.go

```diff
@@ -15,20 +15,42 @@
 package changelog
 
 import (
+	"encoding/json"
+	"fmt"
+
 	"github.com/mongodb/openapi/tools/cli/internal/cli/flag"
 	"github.com/mongodb/openapi/tools/cli/internal/cli/usage"
+	"github.com/mongodb/openapi/tools/cli/internal/openapi"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 )
 
 type Opts struct {
-	fs           afero.Fs
-	basePath     string
-	revisionPath string
-	dryRun       bool
+	fs              afero.Fs
+	basePath        string
+	revisionPath    string
+	exceptionsPaths string
+	dryRun          bool
 }
 
 func (o *Opts) Run() error {
+	changelog, err := openapi.NewChangelog(
```

Comment:
[nit] could we clean up test code and add e2e for the command?

Reply:
the command is not doing anything beside the normalization, if we add the e2e test now we will need to change it completely once the command create the actual changelog. I think we can wait for e2e tests since the command is not ready